composer.json

```diff
@@ -16,17 +16,17 @@
 		}
 	],
 	"require": {
-		"php": "^5.3.2|~7.0",
-		"symfony/framework-bundle": "~2.3|~3.0"
+		"php": "^5.3.9|~7.0",
```

Comment:
Will this PR release 2.0 of this bundle? If so, what about bumping it to `>=5.5.9`?